src/rgw/rgw_sal_motr.cc

```diff
@@ -1444,6 +1444,22 @@ int MotrObject::set_obj_attrs(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx,
       ldpp_dout(dpp, 0) <<__func__<< ": Failed to get key or object's entry from bucket index. rc= " << rc << dendl;
       return rc;
     }
+  // set attributes present in setattrs
+  if (setattrs != nullptr){
+    for (auto& it : *setattrs){
+      attrs[it.first]=it.second;
+    }
+  }
+
+  // delete attributes present in delattrs
+  if (delattrs != nullptr){
+    for (auto& it: *delattrs){
+      auto del_it = attrs.find(it.first);
+        if (del_it != attrs.end()) {
+          attrs.erase(del_it);
```

Comment:
guess this would be needed in cases like deleteObjTagging, to delete tags for an obj. Though if that API is working, seems it may not be through this function, so this code would be useful for some other feature

Reply:
> Yes this will be used to delete an attribute of motr object.

Yes thats what i meant through my comment
> guess this would be needed in cases like deleteObjTagging, to delete tags for an obj.